expression/expression.go

```diff
@@ -91,6 +93,38 @@ func EvalBool(expr Expression, row []types.Datum, ctx context.Context) (bool, er
 	return i != 0, nil
 }
 
+func evalExpr(expr Expression, tp types.TypeClass, row []types.Datum, sc *variable.StatementContext) (interface{}, bool, error) {
```

Comment:
Add comments about those functions that are temporarily used, and it should be removed in the future.